app/controllers/orders_controller.rb

```diff
@@ -3,6 +3,12 @@ def index
     current_user ? @orders = current_user.orders : @orders = {}
   end
 
+  def new
+    if session[:user_id].nil?
+      redirect_to '/login'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.